### PR TITLE
Tests - Fix errors in functional tests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Close sfToolBar, update test description and fix eslint errors in functional tests
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | no tests needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16681)

### CODE REVIEW

File: tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ describe('Create, Read, Update and Delete Customer in BO', async () => {
       this.pageObjects.boBasePage.customersParentLink,
       this.pageObjects.boBasePage.customersLink,
     );
+    await this.pageObjects.boBasePage.closeSfToolBar();
```
Comment (PierreRambaud) [inline tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js:58]: Maybe add the global capability to check if the Symfony toolbar should be closed. Or run test in production env only?

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js:58]: We are already trying to execute tests in production env with `PS_DEV_MODE=0` set in .env file for docker, but while testing it, i got a screenshot with the sf toolbar (so i'm confused).
Maybe we are not doing it the right way.

Comment (PierreRambaud) [reply tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js:58]: It's maybe a problem with the docker image :/

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/customers/customers/02_CRUDCustomerInBO.js:58]: For now, we need this to close sf toolbar, and maybe to keep nightly results green (hope So).

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**